lib/Zonemaster/Backend/Config.pm

```diff
@@ -610,6 +638,8 @@ UNITCHECK {
     _create_setter( '_set_ZONEMASTER_number_of_processes_for_frontend_testing', '_ZONEMASTER_number_of_processes_for_frontend_testing', \&untaint_strictly_positive_int );
     _create_setter( '_set_ZONEMASTER_number_of_processes_for_batch_testing',    '_ZONEMASTER_number_of_processes_for_batch_testing',    \&untaint_non_negative_int );
     _create_setter( '_set_ZONEMASTER_age_reuse_previous_test',                  '_ZONEMASTER_age_reuse_previous_test',                  \&untaint_strictly_positive_int );
+    _create_setter( '_set_RPCAPI_enable_add_api_user',                          '_RPCAPI_enable_add_api_user',                          \&untaint_bool );
+    _create_setter( '_set_RPCAPI_enable_add_batch_job',                         '_RPCAPI_enable_add_batch_job',                         \&untaint_bool );
 }
```

Comment:
Is this equivalent to the following code in #850?
```
sub _set_RPCAPI_enable_add_api_user {
    my ( $self, $value ) = @_;

    $value = untaint_bool( $value ) // die "Invalid value for RPCAPI.enable_add_api_user: $value\n";
    $self->{_RPCAPI_enable_add_api_user} = $value;
    return;
}

sub _set_RPCAPI_enable_add_batch_job {
    my ( $self, $value ) = @_;

    $value = untaint_bool( $value ) // die "Invalid value for RPCAPI.enable_add_batch_job: $value\n";
    $self->{_RPCAPI_enable_add_batch_job} = $value;
    return;
}
```

Reply:
Any comment here?

Reply:
yes it is